pkg/ibmvpc/vpc.go

```diff
@@ -66,20 +66,21 @@ type PrivateIP struct {
 	block *ipblock.IPBlock
 }
 
-func (pip *PrivateIP) Name() string {
+func (pip *PrivateIP) NameForAnalyzerOut(c *vpcmodel.VPCConfig) string {
+	prefix := ""
+	if c != nil && c.IsMultipleVPCsConfig {
+		prefix = pip.VPC().Name() + vpcmodel.Deliminator
+	}
```

Comment:
this code seems to be duplicated across this file.. can be moved to some other internal function that gets config and VPC name as inputs?